Kanvas.podspec.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Kanvas",
+  "version": "1.0.0",
+  "summary": "A camera and media editor.",
+  "homepage": "https://github.com/tumblr/kanvas-ios",
+  "license": {
+    "type": "MPLv2",
+    "file": "LICENSE"
+  },
+  "source": {
+    "git": "https://github.com/tumblr/kanvas-ios.git",
+    "branch": "main"
```

Comment:
@bjtitus are we going to use tag instead of branch?
```
"source": {
    "git": "https://github.com/tumblr/kanvas-ios.git",
    "tag": "1.0.0"
  }
  ```